src/emulator/auth/state.ts

```diff
@@ -597,36 +597,57 @@ export class AgentProjectState extends ProjectState {
     this._usageMode = usageMode;
   }
 
-  // TODO(lisajian): Fill in when v2.projects.tenants.get is added
-  getTenant(): void {
-    throw new NotImplementedError("getTenant is not implemented yet.");
+  getTenantProject(tenantId: string): TenantProjectState {
+    assert(this.tenantProjectForTenantId.has(tenantId), "TENANT_NOT_FOUND");
```

Comment:
Discussed offline during emulator sync - should create tenant if not present. Will include in next revision